transform/transformers/ImageTransformer.py

```diff
@@ -91,12 +91,16 @@ def create_image_sequence(self, path, nmbr_seq=None):
             os.rename(imageFile, fp)
             yield fp
 
-    def create_image_index(self, images, current_time=datetime.datetime.utcnow()):
+    def create_image_index(self, images, current_time=None):
         '''
         Takes a list of images and creates a index csv from them
         '''
         if not images:
             return None
+
+        if current_time is None:
```

Comment:
Do we need an if when it's always going to be done?

Reply:
It won't always be none, None is the default it is overridden by the unit tests

Reply:
👍